shell/apps/utils/ListGen.html

```diff
@@ -0,0 +1,31 @@
+<pre></pre>
```

Comment:
Just making sure you want this and Manifester in this PR, I support checking them in as an experimental thing sooner than later, FWIW.

Reply:
Yes is on purpose. 

If you would prefer I do these things more discretely I can, but I figured nobody would much care in the fullness of time.